docs/developer-access-policy.md

```diff
@@ -0,0 +1,100 @@
+# Policy for OpenSAFELY Access by Platform Developers
+
+## Introduction
+
+OpenSAFELY securely mediates access to pseudonymized patient data for the purpose of analysis (be that research, service evaluation or audit).
+Our objective is to minimise direct access to pseudonymised data wherever possible, to protect privacy;
+while facilitating high quality, efficient analytics on the data by a wide range of users.
+A small number of OpenSAFELY team members need more direct access to the raw pseudonymised data inside the secure environment
+for the purposes of data curation and for developing and running the OpenSAFELY platform.
+
+This policy describes the conditions under which individuals with these roles may access the OpenSAFELY platform
+and the controls that are in place to ensure that such access cannot be used to subvert the OpenSAFELY security model.
+All such access conforms to the OpenSAFELY platform’s principles of security, transparency and minimal access.
+
+The specific types of access activities this policy describes are:
+
+* Running OpenSAFELY test pipelines
+* Ad hoc access to backend systems
+* Data curation activities
+
+## Running OpenSAFELY test pipelines
+
+### Why
+
+For software maintenance and service provision, it is sometimes necessary to execute OpenSAFELY [study pipelines](https://docs.opensafely.org/actions-pipelines/#project-pipelines).
```

Comment:
suggestion:

The link pointed to has a heading of "project pipelines". But the link text is "study pipelines". I'd personally go for consistency and use "project pipelines" here. [^1]

[^1]: If the correct term really is "study pipelines", we should change that everywhere later.

Reply:
comment:

This sentence sounds like it is now talking about both normal running — "service provision" — of OpenSAFELY and testing. Perhaps it's simpler to solely focus on the "software maintenance" aspect? That is what is being discussed. This means you can rewrite as:

> For software maintenance purposes, it is sometimes necessary to run OpenSAFELY project pipelines.